### ISSUE TITLE
Cannot distinguish between hilighted lines

### ISSUE THREAD

**Issue by stratosgear** [jesseduffield/lazygit#744]
**Describe the bug**
I cannot get to have the individual lines get hilighted.

**To Reproduce**
Only way to accurately describe it is through this asciinema recording:

 https://asciinema.org/a/8BhVwa4al8KnPwfJi1pZxMSqm

Without being able to know which file is highlighted it is difficult to know what I'm looking at...

What kind of further info would you like to help you debug this?

```
$ uname -a                                    
Linux serenity 5.5.11-arch1-1 #1 SMP PREEMPT Sun, 22 Mar 2020 16:33:15 +0000 x86_64 GNU/Linux

$ lazygit --version
commit=c5ff39ba5f1331b45a868dd42bbfda8f972e4586, build date=2020-03-21T17:24:32Z, build source=binaryRelease, version=0.16.2, os=linux, arch=amd64
```

**UPDATE**
Now that I see my recorded asciinema, I can see a **very** slight color variation while I am selecting different files.  But I cannot see **anything** on my computer.

**Comment by jesseduffield** [comment]
@stratosgear in other applications are you able to see differences in bold and non-bold text? If not, the issue is that your terminal doesn't distinguish between bold and not bold. It sounds, however, like we need a config option to allow the background color of highlighted lines to change, so that it can be easier to see for people in your situation.

**Comment by stratosgear** [comment]
Yes, I do not have any issues with other programs (none that I have identified so far, at least, and **I am** a heavy terminal user too).

Indeed, if there was an option to configure the background (and maybe text color) of the active item, that would be sweet!

Congrats on a really useful app, too!

**Comment by band-a-prend** [comment]
> It sounds, however, like we need a config option to allow the background color of highlighted lines to change, so that it can be easier to see for people in your situation.

+1

@jesseduffield 
As I mentioned in https://github.com/jesseduffield/lazygit/issues/734#issuecomment-610662866 I also have trouble with distinguish selected item as in my case I use bold font in terminal.

**Comment by deb-master** [comment]
Same here (using bold JetBrains Mono in xterm), the selection is impossible to distinguish.

**Comment by jesseduffield** [comment]
This is fixed in release v0.20.3 :) (thanks to @quantonganh for investigation). If you want things to be really visible I recommend the following config:

```
  gui:
    theme:
      selectedLineBgColor:
        - reverse
```
